File: src/main/java/io/github/dqualizer/dqlang/types/adapter/constants/TechnicalConstants.java
package io.github.dqualizer.dqlang.types.adapter.constants;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.ToString;

@Getter
@ToString
public class TechnicalConstants {

    @JsonProperty("warm_up_duration")
    private int warmUpDuration;
    @JsonProperty("cool_down_duration")
    private int coolDownDuration;
    @JsonProperty("think_time")
    private int thinkTime;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/types/adapter/constants/loadTest/TechnicalConstants.kt
package io.github.dqualizer.dqlang.types.adapter.constants.loadTest

import com.fasterxml.jackson.annotation.JsonProperty

data class TechnicalConstants(
  @JsonProperty("warm_up_duration")
  val warmUpDuration: Int,

  @JsonProperty("cool_down_duration")
  val coolDownDuration: Int,

  @JsonProperty("think_time")
  val thinkTime: Int
)
